.github/workflows/gh-actions.yml

```diff
@@ -15,7 +15,13 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - name: Initializtaion
-        run: sudo apt-get install doxygen vera++ cppcheck pylint pylint python-serial -q
+        run: |
+          sudo apt-get install doxygen vera++ cppcheck -q
+          sudo pip install pyserial pylint
```

Comment:
Whether python packages are installed via apt or pip is a matter of taste. I wouldn't call apt-based installation as improper -- as long as it works. And now that #4388 has landed, apt should be working again all right.

Reply:
Oh, I've done that because on Ubuntu 20.04, apt-get install python-serial would fail, pip install are more stable.